--- commerce/modules/cart/commerce_cart.module	(revision )
+++ commerce/modules/cart/commerce_cart.module	(revision )
@@ -1339,7 +1339,8 @@
 
     // Allow other modules to specify what properties should be compared when
     // determining whether or not to combine line items.
-    drupal_alter('commerce_cart_product_comparison_properties', $comparison_properties, clone($line_item));
+    $line_item_clone = clone($line_item);
+    drupal_alter('commerce_cart_product_comparison_properties', $comparison_properties, $line_item_clone);
 
     // Loop over each line item on the order.
     foreach ($order_wrapper->commerce_line_items as $delta => $matching_line_item_wrapper) {
